Packs/AutoFocus/Integrations/AutoFocusTagsFeed/AutoFocusTagsFeed.py

```diff
@@ -0,0 +1,509 @@
+        for ref in refs:
+            url = ref.get('url')
+            source = ref.get('source')
+            time_stamp = ref.get('created')
+            title = ref.get('title')
+            fields['publications'].append({'link': url, 'title': title, 'source': source, 'timestamp': time_stamp})
+    fields['aliases'] = tag_details.get('aliases')
+    fields['description'] = tag.get('description')
+    last_hit = tag.get('lasthit')
+    fields['lastseenbysource'] = datetime.strptime(last_hit, AF_TAGS_DATE_FORMAT).strftime(
+        DATE_FORMAT) if last_hit else None
+    updated_at = tag.get('updated_at')
+    fields['updateddate'] = datetime.strptime(updated_at, AF_TAGS_DATE_FORMAT).strftime(
+        DATE_FORMAT) if updated_at else None
+    fields['reportedby'] = tag.get('source')
+    remove_nulls_from_dictionary(fields)
+    return fields
+
+
+def create_relationships_for_tag(client: Client, name: str, tag_type: str, related_tags: List[str]):
+    """
+    Creates all the relationships of an indicator.
+    Args:
+        client: Client class
+        name: The indicator's name
+        tag_type: The indicator's type
+        related_tags: A list of all indicators related to the spesific indicator
+    Returns:
+        a list represents the relationships of an indicator.
+    """
+    relationships: list = []
+    for related_tag in related_tags:
+        try:
+            related_tag_details = client.get_tag_details(related_tag)
+        except DemistoException:
+            demisto.debug(f'AutoFocus Tags Feed: Could not create relationship for {name} with {related_tag}.')
+            continue
+        if related_tag_details:
+            tag = related_tag_details.get('tag')
+            related_tag_name = tag.get('tag_name')
+            tag_class = tag.get('tag_class')
+            source = tag.get('source')
+            related_tag_type = get_tag_class(tag_class, source)
+            if related_tag_type:
+                relationships.append(
+                    create_relationship(name, tag_type, related_tag_name, related_tag_type).to_indicator())
+    return relationships
+
+
+def create_relationship(a_name: str, a_class: str, b_name: str, b_class: str):
+    return EntityRelationship(
+        name=MAP_RELATIONSHIPS.get(a_class, {}).get(b_class),
+        entity_a=a_name,
+        entity_a_type=a_class,
+        entity_b=b_name,
+        entity_b_type=b_class,
+        reverse_name=MAP_RELATIONSHIPS.get(b_class, {}).get(a_class),
+    )
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client) -> str:
+    """Builds the iterator to check that the feed is accessible.
+    Args:
+        client: Client object.
+    Returns:
+        Outputs.
+    """
+    try:
+        response = client.get_tags(data={'pageSize': 1})
+        tag_list = response.get('tags', [])
+        tag = tag_list[0] if len(tag_list) > 0 else {}
+        tag_name = tag.get('public_tag_name')
+        client.get_tag_details(tag_name)
```

Comment:
Isn't the first call enough to show a successful connection? why do we need the second one?